src/iosrv/drpc_progress.c

```diff
@@ -371,7 +371,7 @@ handle_incoming_call(struct drpc *session_ctx)
 	 * Ownership of the call context is passed on to the handler ULT.
 	 */
 	rc = dss_ult_create(drpc_handler_ult, (void *)call_ctx,
-			    DSS_ULT_DRPC_HANDLER, 0, 0, NULL);
+			    DSS_XS_SYS, 0, 0, NULL);
```

Comment:
I forgot why drpc handler executes on xs 0 that different with drpc listener (xs 1), does it need to send cart RPC in drpc handler? but unrelated with this patch.